scalameta/semanticdb-scalac/src/main/scala/scala/meta/internal/SemanticdbPipeline.scala

```diff
@@ -98,7 +111,37 @@ trait SemanticdbPipeline extends DatabaseOps { self: SemanticdbPlugin =>
           cleanupUpwards(ve.uri.toFile)
         })
         super.run()
+        timestampPersistFinished = System.nanoTime()
+        reportSemanticdbSummary()
       }
     }
   }
+
+  private val timestampPluginCreated = System.nanoTime()
```

Comment:
Did you consider using scalac Statistics instead of rolling it yourself? I'm fine either way, just a note.

Reply:
These are changes from a PR I picked up and thought you had already approved.  I can’t comment on the reasoning behind them.

Reply:
Didn't know much about `Statistics`, to be honest. I always thought it's something hacked up, but turns out it's not. Let's keep the existing implementation though and refactor if we add more performance counters.